tests/src/test/scala/org/http4s/MessageSpec.scala

```diff
@@ -14,27 +15,28 @@ class MessageSpec extends Http4sSpec {
       val remote = InetSocketAddress.createUnresolved("www.remote.com", 45444)
 
       "get remote connection info when present" in {
-        val r = Request().withAttribute(
-          Request.Keys.ConnectionInfo(Request.Connection(local, remote, false)))
+        val r = Request[IO]().withAttributeEntry(
+          Request.Keys.ConnectionInfo(Request.Connection(local, remote, false))
+        )
         r.server must beSome(local)
         r.remote must beSome(remote)
       }
 
       "not contain remote connection info when not present" in {
-        val r = Request()
+        val r = Request[IO]()
         r.server must beNone
         r.remote must beNone
       }
 
       "be utilized to determine the address of server and remote" in {
-        val r = Request().withAttribute(
+        val r = Request[IO]().withAttributeEntry(
```

Comment:
`withAttribute` overload needed to be switch out with a specific method so `withAttributeEntry`